microsoft/testsuites/core/cpu.py

```diff
@@ -1,26 +1,55 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT license.
 
+import re
+import time
 from pathlib import PurePosixPath
+from typing import List
 
 from assertpy.assertpy import assert_that
 
-from lisa import (
-    BadEnvironmentStateException,
-    LisaException,
-    Logger,
-    Node,
-    SkippedException,
-    testsuite,
-)
-from lisa.tools import Cat, Echo, Lscpu, Lsvmbus, Uname
+from lisa import Logger, testsuite
+from lisa.base_tools.cat import Cat
+from lisa.base_tools.uname import Uname
+from lisa.node import Node
+from lisa.tools.echo import Echo
+from lisa.tools.lscpu import Lscpu
+from lisa.tools.lsvmbus import Lsvmbus
+from lisa.tools.taskset import TaskSet
+from lisa.util import BadEnvironmentStateException, LisaException, SkippedException
 
 
 class CPUState:
     OFFLINE: str = "0"
     ONLINE: str = "1"
 
 
+hyperv_interrupt_substr = ["hyperv", "Hypervisor", "Hyper-V"]
+
+
+class Interrupt:
```

Comment:
Add dataclass decorator. If not, the members are class variable, and can only be shared cross classes.